open-science-projects.md

```diff
@@ -1,49 +1,102 @@
 -	It is easier to learn when resources are available to refer back to
--	Open-Science is like a group project. Except the world is your group and they won’t affect your grade. 
--	Documenting your work helps you, aids others and helps them help you 
+-	Open-Science is like a group project. Except the world is your group and they won’t affect your grade.
+-	Documenting your work helps you, aids others and helps them help you
+
+
+# The day-to-day supervisor perspective
+
+As an early-stage researcher, it can often be difficult to find opportunities to develop and practice coaching, teaching and supervisory skills alongside making progress on your own workload and helping contribute to the lab's goals. Likewise, as someone heavily involved on the riboviz project as a developer, it often seems like there are so many useful new features or improvements we'd like to add, and never enough time to get them done. Enter: undergraduate open science thesis projects!
+
+## Overlapping goals
+Supervising undergraduate students working on riboviz-related research projects in a day-to-day capacity along with my PI has presented an excellent opportunity for me to build key skills, as well as aligning the supervision with relevant work of my own, meaning that there's not as great a 'brain switch' required when going between supervision tasks and my standard workload. It also means that I have been able to improve my own knowledge while helping support the students to learn the concepts, technical skills and organisational techniques required to complete their projects successfully.
+
+Having two students joining to pursue projects with riboviz at the same time this year was particularly helpful, as a lot of the instruction and coaching were applicable to both, and meant that there was also an opportunity for peer learning. While not all labs might have capacity to offer multiple projects, I was surprised to find that the increase in supervision time required by two students was much less than expected. Many of the same problems were encountered during both projects and could be solved in one discussion.
+
+Supervising two undergraduate projects at the same time was also unexpectedly valuable in highlighting areas where my instruction, knowledge, or communication skills could be improved: _"When one student doesn't seem to understand, it could go either way. When two students don't seem to understand, the problem probably lies with the communicator!"_ The process has been really helpful in making me re-think how I communicate complex ideas, technical information and scientific concepts, and has contributed significantly to my personal development.
```

Comment:
If this is a quote does it need a citation?

Reply:
I'm quoting me, but this is the first time I've said it before, so I'm not sure what style guides recommend for 'quotes of the future'... :grin: